src/amber/router/params_parser.cr

```diff
@@ -78,7 +78,13 @@ module Amber::Router
     end
 
     def merge_route_params
-      route_params.each { |k, v| params[k] = v }
+      route_params.each do |k, v|
+        if k == route_params.keys.last
```

Comment:
This 2 lines should probably be in its own method, called `format`. I forgot to tell you in the previous PR that we have already define a similar method in the context https://github.com/amberframework/amber/blob/master/src/amber/router/context.cr#L72

Reply:
I `format` currently being used for something? My need is just to remove 5 ext's from the end of the last path_param. I could extract it to a method but I don't think moving it to context makes sense.

Reply:
If it don't remove these then paths like `/pets/1.json` try looking up an id of `1.json` in Granite which fails. This isn't where I'm detecting the format, since it will only sometimes be on a param I have to parse it from the path, for responses.